plugins/catalog-backend/src/service/DefaultEntitiesCatalog.ts

```diff
@@ -379,6 +379,7 @@ export class DefaultEntitiesCatalog implements EntitiesCatalog {
               facet.substring('metadata.labels.'.length)
             ];
           }
+          if (facet === 'owner') return lodash.get(entity.spec, facet);
```

Comment:
This line should be removed.

If you want to read from the spec then you have the option of using `spec.owner` as the facet key. Any json path basically within objects can be specified; see `lodash.get` on the next line.

But more importantly, readers just should never get relations from the spec. It just doesn't work well at all:

- There can be any number of sources for relations, including fetching them dynamically from third party services / data stores / codeowners file / phase of the moon / whatnot
- Thus there may actually be any number of relations of any type, even if the spec only leaves room for one
- The spec field has complex behaviors in terms of what namespace and kind end up being used in the actual relation: the owner may be set either as `foo`, or `group:foo`, or `default/foo`, or `group:default/foo`, or `FOO`, or ... etc, but the end result is always `group:default/foo` in the generated relation.